src/connectors/primephonic.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+Connector.playerSelector = '.player';
+
+Connector.trackArtSelector = '#currently-playing-album-cover img';
+
+Connector.artistSelector = '#currently-playing-composer';
+
+Connector.trackSelector = '.currently-playing .movement';
+
+Connector.albumSelector = '#currently-playing-work';
+
+Connector.currentTimeSelector = '#time_container_current';
+
+Connector.getRemainingTime = () => {
```

Comment:
`Connector.getRemainingTime` is used to calculate current time based on duration. You have no duration getter or selector, so this function is useless in this context.

https://github.com/web-scrobbler/web-scrobbler/blob/master/src/core/content/connector.js#L358

Reply:
Ah thanks, I'll look at that again. 
By default Primephonic doesn't show the total time, only time remaining, so I might just remove it for the first version.